src/components/Breadcrumbs/Breadcrumbs.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { withRouter, Link } from "react-router-dom";
+
+const Breadcrumbs = (props) => {
+  const {
+    history,
+    location: { pathname },
+  } = props;
+  const pathnames = pathname.split("/").filter((x) => x);
+  return (
+    <div className="avenir pa2 ph5 link flex">
+      {pathnames.length > 0 && (
+        <div>
+          <Link
+            className="no-underline avenirlink hover-green hover-animate"
```

Comment:
avenirlink -> avenir link

Reply:
oops, theres no such thing is avenirlink,, the space must have been lost somewhere along the way!